security/gopass/Portfile

```diff
@@ -15,356 +15,12 @@ homepage            https://www.gopass.pw
-                        size    8633 \
-                    github.com/caspr-io/yamlpath \
-                        lock    502e8d113a9b \
-                        rmd160  7b3d05122f821aac68278a797aa205b09312d25a \
-                        sha256  4975c9a92c5f4eb1034e916cdea2aaa96dd912dabc8f7e6379900a1a6e579d58 \
-                        size    6334 \
-                    github.com/boombuler/barcode \
-                        lock    v1.0.1 \
-                        rmd160  fe7247722491a03f56b9a41ae144ab05487f29fc \
-                        sha256  bd623e3ea8d79f74464b9ee4f16808545842b0e2c06b07e9b96e2dc5df80e40f \
-                        size    62985 \
-                    github.com/blang/semver \
-                        lock    v4.0.0 \
-                        rmd160  ab0e0d974dcbc784840d4bcc76584242436c51fa \
-                        sha256  bf4d622c039173a4e0903738d8b5c788c3c63bb8cfa7485611f38aece3504d0f \
-                        size    27781 \
-                    github.com/atotto/clipboard \
-                        lock    v0.1.4 \
-                        rmd160  cda277fa418bc6cdb42b3a2e6c3637473bdd12e3 \
-                        sha256  6d474bab7ef589dd95a56d6fd571d447fdfbcc8c1985b7b4841cfa98edc0a97f \
-                        size    5023 \
-                    github.com/alessio/shellescape \
-                        lock    v1.4.1 \
-                        rmd160  669d9a26af060752e544e09551b8835dc89b336a \
-                        sha256  ab26daceb19d4973dd1bf37105473f8d27697b5a04b124d790fd45124450b9a2 \
-                        size    7552 \
-                    github.com/ProtonMail/go-crypto \
-                        lock    cf6655e29de4 \
-                        rmd160  0c878f2d4a5a2e8869df40d99ee219bdff7503de \
-                        sha256  5b94489154983036fbaf619202a3f92700817fd57b3a1010f0a39bf2e2432665 \
-                        size    330467 \
-                    filippo.io/edwards25519 \
-                        repo    github.com/FiloSottile/edwards25519 \
-                        lock    v1.0.0 \
-                        rmd160  47330d4bd65ec5e115038359a989a1a3f775e130 \
-                        sha256  f9474496e781cc9985cc575fbd108a5ca1992cc9fdfd35ee0efcb2818fab928c \
-                        size    39898 \
-                    filippo.io/age \
-                        repo    github.com/FiloSottile/age \
-                        lock    v1.0.0 \
-                        rmd160  3a01840612afa65c6dc7fa897b2dc573ed869da6 \
-                        sha256  30d2ab91b9f13ca4aa2c079ca688013658965e827557aa461296932d633c4055 \
-                        size    59693
+checksums           rmd160  336a9f37e55eca86db312822f5cc1ca31189fdc4 \
+                    sha256  4e36214e05fe51fa6e253c0640a1827c27dec29552695c29c710bdb3ac5292cb \
+                    size    2304242
 
 set go_ldflags      "-s -w -X main.version=${version}"
+build.env-delete    GO111MODULE=off GOPROXY=off
```

Comment:
Is it some new/recommended way to package go apps without using `go.vendors`?

Reply:
No, this is not recommended, but it's the workaround when it's not possible to build using `go.vendors`. In this case, `go.vendors` isn't working for this latest version because one of the require Go packages is at a particular domain that isn't supported by `go.vendors`.